mobileApp/lib/ui/IntroFlow/providers.dart

```diff
@@ -0,0 +1,42 @@
+// 🐦 Flutter imports:
+import 'package:flutter/material.dart';
+
+// 📦 Package imports:
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:rxdart/rxdart.dart';
+
+// 🌎 Project imports:
+import 'package:YouOweMe/resources/providers.dart';
+import 'package:YouOweMe/ui/IntroFlow/loginUser.dart';
+
+final introFlowPageControllerProvider =
+    Provider((ref) => PageController(initialPage: 0));
+
+final authValidatorProvider = StreamProvider<bool>((ref) {
+  BehaviorSubject<bool> screeningSubject = BehaviorSubject.seeded(false);
```

Comment:
Maybe Test this with a null and check if the Loading page show's up, and see how it looks.